modules/pam_succeed_if/pam_succeed_if.c

```diff
@@ -292,7 +292,7 @@ evaluate(pam_handle_t *pamh, int debug,
 	 const char *left, const char *qual, const char *right,
 	 struct passwd **pwd, const char *user)
 {
-	char buf[LINE_MAX] = "";
+	char buf[21] = "";
```

Comment:
I understand `21` is `sizeof(long) * log10(256) + 1`, but could this be written explicitly?
Given that we do not have to save a few bytes, `3` is an acceptable approximation of `log10(256)`.

I'd probably also renamed `buf` to something saying it contains a string representation of a number.

Reply:
Adjusted